server/model/viewModel.ts

```diff
@@ -11,7 +12,9 @@ export default class ViewModel {
     allAdjustments: Adjustment[],
     public adjustmentType: AdjustmentType,
   ) {
-    this.adjustments = allAdjustments.filter(it => it.adjustmentType === adjustmentType.value)
+    this.adjustments = allAdjustments
+      .filter(it => it.adjustmentType === adjustmentType.value)
```

Comment:
the ticket specifies the sort order in the table to be based on the from date for UAL's... this sort applies to all adjustment types though. Will communicate to analyst